content/webapp/components/ImageGallery/ImageGallery.tsx

```diff
@@ -231,14 +231,13 @@ const ButtonContainer = styled.div<{ isHidden: boolean }>`
   z-index: 2;
 `;
 
-type Props = {
-  id: string;
+export type Props = {
   title?: string;
   items: CaptionedImageProps[];
   isStandalone: boolean;
 };
 
-const ImageGallery: FunctionComponent<Props> = ({
+const ImageGallery: FunctionComponent<{ id: string } & Props> = ({
```

Comment:
Why the move off `Props` into here?

Reply:
The `id` is only populated when we create the component in Body.tsx; the transformEditorialImageGallerySlice method doesn't return an id.